### DISCUSSION TITLE
「开源之书·共读」#38：群体的智慧 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#87] [Announcements]
# 「开源之书·共读」#38：群体的智慧 - 「开源之道」

https://opensourceway.community/posts/book-of-open-source/38-the-wisdom-of-crowds/

**Comment by ZoomQuiet** [comment]
#是也乎

都是广告, 其实吧, 群体智慧, 也就是 MEME,
也就是社会共识,
以往整个时代只有少数共识,
现在高速文化, 好在还是只有少数基础共识, 是永远不会变的;

开源, 就是回归简单共识: talk is cheap , show me code.
然后, 才有了长期可持续的积累,
最后, 才有聪明的职业经理人, 冲进来变成大生意而已...
